src/m365/pa/commands/environment/environment-get.spec.ts

```diff
@@ -80,15 +80,15 @@ describe(commands.ENVIRONMENT_GET, () => {
       return Promise.reject('Invalid request');
     });
 
-    await command.action(logger, { options: { debug: true, name: 'Default-d87a7535-dd31-4437-bfe1-95340acd55c5' } });
+    await command.action(logger, { options: { debug: true, varbose: true } });
```

Comment:
Let's specify either `debug` or `verbose`. Because when `debug` is specified, `verbose` will also be `true`. I prefer `verbose` in tests.